serverlessworkflow/modules/ROOT/pages/eventing/handling-events-on-workflows.adoc

```diff
@@ -1 +1,150 @@
+
+`exclusive` property is used to determine if there is an event match. If `exclusive` value is false, all CloudEvent types in `eventRefs` array should be received for a match to occur. Otherwise, the reception of any of the referenced CloudEvent types is consider a match. 
+
+=== Event match example
+
+Lets illustrate the usage of previously mentioned fields with an example
+
+You are going to use two CloudEvent Types: `noisy` and `silent`
+
+[source, json]
+----
+"events": [
+    {
+      "name": "noisyEvent",
+      "source": "",
+      "type": "noisy",
+      "dataOnly" : "false"
+    },
+    {
+      "name": "silentEvent",
+      "source": "",
+      "type": "silent"
+    }
+  ]
+----
+
+You want to indicate there is a match only when both a `noisy` and a `silent` CloudEvent types are received. You also want to execute a specific action for `noisy` and a different one for `silent`. You will indicate that by defining an Event state containing both event definitions in separate `onEvent` items and  setting `exclusive` to `false`.
+
+[source, json]
+----
+  {
+    "name": "waitForEvent",
+    "type": "event",
+    "onEvents": [
+      {
+        "eventRefs": [
+          "noisyEvent"
+         ],
+         "actions": [ 
+           {
+             "functionRef": "letsGetLoud"
+           }
+         ]
+      },
+      {
+        "eventRefs": [
+           "silentEvent"
+        ],
+        "actions": [ 
+          {
+            "functionRef": "beQuiet"
+          }
+        ]
+      }
+    ]
+    , 
+    "exclusive": false
+  }
+----
+
+If you want to execute the same action when both event types are received, you use just one `onEvent` item
+
+[source, json]
+----
+  {
+    "name": "waitForEvent",
+    "type": "event",
+    "onEvents": [
+      {
+        "eventRefs": [
+          "noisyEvent",
+          "silentEvent"
+         ],
+         "actions": [ 
+           {
+             "functionRef": "notCareAboutNoise"
+           }
+         ]
+       }
+    ]
+    ,
+    "exclusive": false
+  }
+----
+
+If you want the same action to be executed when any of the event types are received, you set `exclusive` to true. You can also remove `exclusive`, since `true is the default value. 
+
+[source, json]
+----
+  {
+    "name": "waitForEvent",
+    "type": "event",
+    "onEvents": [
+      {
+        "eventRefs": [
+          "noisyEvent",
+          "silentEvent"
+         ],
+         "actions": [ 
+           {
+             "functionRef": "notCareAboutNoise"
+           }
+         ]
+       }
+    ]
+    ,"exclusive": true
+  }
+----
+
+
+== Starting a workflow instance
+
+An event state can be used to start a new workflow instance. To enable this functionality, the `start` property in the workflow header should be set to the event state name. The workflow instance will be started when there is a an event match. 
+
+Given the previously defined `waitForEvent` state, you set as starting event by adding following code at workflow header. 
```

Comment:
I think when starting a new instance, the `"exclusive": false`, I mean waiting to receive more than one event to start a new instance...

Reply:
exclusive false means that you have to wait for all other event types in eventRef array before triggering the action of the event: start or resume.